examples/paper_examples/sn2_explicit/train.py

```diff
@@ -0,0 +1,82 @@
+import gaptrain as gt
+gt.GTConfig.n_cores = 10
+
+
+def train_h2o():
+    """Train the intramolecular PES of H2O"""
+    _, _ = gt.active.train(h2o,
+                           gap=gt.GAP(name='intra_h2o', system=h2o),
+                           method_name='gpaw',
+                           validate=False,
+                           temp=1000)
+    return None
+
+
+def train_sn2():
+    """Train the intramolecular PES of a Cl+CH3Cl reaction"""
+    _, _ = gt.active.train(sn2,
+                           gap=gt.GAP(name='intra_sn2', system=sn2),
+                           active_e_thresh=0.1,
+                           max_time_active_fs=500,
+                           method_name='gpaw',
+                           validate=False,
+                           temp=500)
+    return None
+
+
+def train_inter():
+    """Train the intermolecular components between H2O and the reaction"""
+    _, _ = gt.active.train(system,
+                           method_name='gpaw',
+                           validate=False,
+                           fix_init_config=True,
+                           active_e_thresh=0.5,
+                           gap=gt.gap.SSGAP(solute_intra=intra_sn2_gap,
+                                            solvent_intra=intra_h2o_gap,
+                                            inter=inter_gap))
+    return None
+
+
+if __name__ == '__main__':
+
+    h2o = gt.System(box_size=[10, 10, 10])
+    h2o.add_solvent('h2o', n=1)
+    train_h2o()
+
+    sn2 = gt.System(box_size=[10, 10, 10])
+    sn2.add_molecules(gt.Molecule('ts.xyz'))
```

Comment:
47: do you need to specify the charge here?

Reply:
also needs   ` fix_init_config=True` in train_sn2() – whoops. thanks 👍